script/upload_to_rpm_repo

```diff
@@ -14,32 +14,37 @@ set -exu
 #
 # Date          Channel       Fingerprint           Full Key
 #
+# 10/17/2019    Release       0xA8580BDC82D3DC6C    9BD198A2D848C482E3550697A8580BDC82D3DC6C
```

Comment:
Why is the KEY_ID changing? I don't think we're ready to start signing with the new subkey yet, it hasn't made it to users' machines yet.

Reply:
This could list both the old and new key for a time too, if we wanted to add another column.

Reply:
This is just a list of keys. It's a reference, but not used anywhere. As an alternative we could simply provide a link to the wiki where another table of these keys is kept.

Reply:
Resolved by moving the subkey from the list (since it only references subkeys)